src/transport/mod.rs

```diff
@@ -134,6 +134,28 @@ pub fn accept(acceptor: &TcpListener) -> IoResult<Transport> {
     })
 }
 
+// FIXME: There needs to be a way to break from this blocking command.
+// (Though this seems to be impossible with the current Rust TCP API).
+#[cfg(test)]
+pub fn accept_utp(acceptor: &::utp::UtpListener) -> IoResult<Transport> {
+    let (stream, _remote_endpoint) = try!(acceptor.accept()
+                                          .map_err(|e| io::Error::new(io::ErrorKind::NotConnected, e.description())));
```

Comment:
A lot of these changed in new crust code so soon we need proper error types that can be passed up. So the string type errors will become less used in the code.